nf_core/pipeline-template/conf/test_full.config

```diff
@@ -22,3 +22,9 @@ params {
   // Ignore `--input` as otherwise the parameter validation will throw an error
   schema_ignore_params = 'genomes,input_paths,input'
 }
+
+// Overwrite trace reports etc - allows '-resume' on AWS
+timeline.overwrite = true
+report.overwrite = true
+trace.overwrite = true
+dag.overwrite = true
```

Comment:
```suggestion
def timeStamp = new java.util.Date().format( 'yyyy-MM-dd_HH:mm:ss')
timeline.file = "${params.tracedir}/execution_timeline_${timeStamp}.html"
report.file    = "${params.tracedir}/execution_report_${timeStamp}.html"
trace.file      = "${params.tracedir}/execution_trace_${timeStamp}.txt"
dag.file        = "${params.tracedir}/pipeline_dag_${timeStamp}.svg"
```

Reply:
Is this what you meant?
If we don't care about the file ending it could just be
e.g `trace.file += ".${timeStamp}"`

Reply:
Nice! Yes this is what I meant 👍🏻 I think that we do care about the file endings, as that will mess up the default opening tool etc.

Only minor change that could be nice is to make `timeStamp` a `params` variable so that it can be overwritten / removed if desired. Also not camelCase 👀 😅

Reply:
Oh, and move it into `nextflow.config` so that it's for everyone (as others may also be running on AWS outside of the test configs). I'll do these things now 👍🏻